src/linker/Linker.Steps/UnreachableBlocksOptimizer.cs

```diff
@@ -806,6 +806,18 @@ public bool ApplyTemporaryInlining (in UnreachableBlocksOptimizer optimizer)
 				return changed;
 			}
 
+			static bool IsConditionalBranch (OpCode opCode)
+			{
+				switch (opCode.Code) {
+				case Code.Brfalse:
+				case Code.Brfalse_S:
+				case Code.Brtrue:
+				case Code.Brtrue_S:
+					return true;
+				}
+				return false;
+			}
```

Comment:
nit: you could use new C# syntax and make it one liner

```suggestion
			=> opCode.Code is Code.Brfalse or Code.Brfalse_S or Code.Brtrue or Code.Brtrue_S;
```